src/pages/users/AddUsers.vue

```diff
@@ -233,15 +234,29 @@ const onFileUpload = async (event) => {
   // Parse the file directly with csvFileToJson
   const parsedData = await csvFileToJson(file);
 
-  parsedData.forEach((user) => {
+  // Filter out completely empty rows
+  const filteredData = parsedData.filter((user) => {
+    if (!user || typeof user !== 'object') return false;
+    const keys = Object.keys(user);
+    if (keys.length === 0) return false;
```

Comment:
For readability, I think it makes sense to do this outside of this parsing, because we really only need to do it once. The keys will always equal the columns, so if any columns are missing, then all users would have it missing.